.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "libocispec"]
 	path = libocispec
-	url = https://github.com/containers/libocispec.git
+	url = https://github.com/lifeng68/libocispec.git
```

Comment:
Can we get these changes merged upstream into containers/libocispec.git?

Reply:
There is a PR opened for libocispec and we are working on the last details before merging it

Reply:
@rhatdan @giuseppe Fixed.